Signal/src/ViewControllers/SignalsViewController.m

```diff
@@ -902,6 +906,13 @@ - (void)yapDatabaseModified:(NSNotification *)notification {
     [self checkIfEmptyView];
 }
 
+- (void)setViewingThreadsIn:(CellState)viewingThreadsIn
+{
+    _viewingThreadsIn = viewingThreadsIn;
+
+    [self checkIfEmptyView];
```

Comment:
`viewingThreadsIn` affects `checkIfEmptyView`; we should call `checkIfEmptyView` every time `viewingThreadsIn` changes - especially because the default value is `kArchiveState` not `kInboxState`.